pages/e-lab.tsx

```diff
@@ -1,25 +1,22 @@
+import Benefits from "@components/Benefit";
+import FAQ from "@components/FAQ";
 import Logos from "@components/Logos";
 import Stat from "@components/Stat";
+import Testimonials from "@components/Testimonials";
+import Timeline from "@components/Timeline";
 import Button from "@components/ui/Button";
 import Section from "@components/ui/Section";
-import Image from "next/image";
-import Head from "next/head";
-import Testimonials from "@components/Testimonials";
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import {
   faBook,
   faChevronDown,
   faCircleNodes,
   faHandshakeSimple,
   faHandsHoldingCircle,
 } from "@fortawesome/free-solid-svg-icons";
-import Benefits from "@components/Benefit";
-import Timeline from "@components/Timeline";
-import FAQ from "@components/FAQ";
-import { faq } from "data/e-lab";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```

Comment:
why

Reply:
I swear I didnt do it

Reply:
It was there before, my linter just moved the imports order

Reply:
oh okay